src/Language/PureScript/Sugar/Operators.hs

```diff
@@ -249,9 +250,38 @@ externsFixities ExternsFile{..} =
       , Fixity assoc prec
       , name
       )
+desugarTypeClassTypeOpsForModule
```

Comment:
Also, it's not clear to me that we gain much from separating this into two functions — have you considered just using one function? Presumably we wouldn't need to change `collectFixities` if we only used one function?